api/api.go

```diff
@@ -258,3 +258,20 @@ type FeatureDescriber interface {
 type CsvWriter interface {
 	WriteCsv(context.Context, io.Writer) error
 }
+
+// ChargeMode is the charge operation mode. Valid values are off, now, minpv and pv
+type BatteryMode string
+
+// Charge modes
+const (
+	BatteryUnknown BatteryMode = ""
+	BatteryNormal  BatteryMode = "normal"
+	BatteryLocked  BatteryMode = "locked"
+	BatteryCharge  BatteryMode = "charge"
+)
+
+type BatteryControl interface {
```

Comment:
BatteryController

Muss das Meter inkludieren oder ist das nicht separat?

Reply:
Könnte auch separat, nur brauche ich dann einen anderen Weg um passende Mocks in den Tests zu injizieren. Muß ich dafür ggf. Zwecks Test ein eigenes Mock Interface definieren, oder kann ich auch go mock signalisieren, daß ich gerne einen Mock hätte, der zwei Interfaces (Charger + BatteryController) implementiert?

Reply:
Das ist einfach ein struct mit beiden Interfaces drin- gibts diverse Beispiele in den Loadpoint-Tests

Reply:
Großartig! Ich danke Dir, wieder was gelernt.